modules/common/src/main/scala/protocol.scala

```diff
@@ -37,8 +33,6 @@ sealed abstract class CompressionType extends Product with Serializable
 case object Identity                  extends CompressionType
 case object Gzip                      extends CompressionType
 
-class rpc(val serializationType: SerializationType, val compressionType: CompressionType = Identity)
-    extends StaticAnnotation
 @deprecated("Streaming type is now determined from request and response types", "0.14.0")
```

Comment:
I think we could remove the `stream`, since we are no longer supporting the @rpc annotation